crates/cubecl-core/src/frontend/cmma.rs

```diff
@@ -441,6 +628,80 @@ pub mod cast {
     }
 }
 
+/// Manually execute a low level `mma` operation with manually managed registers. Register layout
+/// and index mapping can be retrieved from the [`MatrixDefinition`]
+#[allow(unused_variables)]
+pub fn execute_manual<AB: CubePrimitive, CD: CubePrimitive>(
+    matrix: &MmaDefinition<AB, CD>,
+    a_registers: &Sequence<Line<AB>>,
+    b_registers: &Sequence<Line<AB>>,
+    c_registers: &Sequence<Line<CD>>,
+    d_registers: &mut Sequence<Line<CD>>,
+) {
+    unexpanded!()
+}
+
+/// Module containing the expand function for [execute_manual()].
+pub mod execute_manual {
+    use crate::prelude::SequenceExpand;
+
+    use super::*;
+
+    /// Expand method of [execute_manual()].
+    #[allow(unused_variables)]
+    pub fn expand<AB: CubePrimitive, CD: CubePrimitive>(
+        scope: &mut Scope,
+        matrix: MmaDefinitionExpand<AB, CD>,
+        a_registers: SequenceExpand<Line<AB>>,
+        b_registers: SequenceExpand<Line<AB>>,
+        c_registers: SequenceExpand<Line<CD>>,
+        d_registers: SequenceExpand<Line<CD>>,
```

Comment:
Is there a reason to use Sequence instead of Array? Both works, just wondering how using sequence is easier (in terms of API).